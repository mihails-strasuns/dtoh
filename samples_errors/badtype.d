module badtype;
class C { }
extern(C) __gshared C var;
