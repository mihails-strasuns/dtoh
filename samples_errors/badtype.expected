samples_errors/badtype.d(3) [badtype.C]
	Type of kind 'class' can't be represented in C
